packetbeat/magefile.go

```diff
@@ -120,7 +102,7 @@ func Package() {
 
 	devtools.UseElasticBeatPackaging()
 	devtools.PackageKibanaDashboardsFromBuildDir()
-	customizePackaging()
+	packetbeat.CustomizePackaging()
 
 	mg.Deps(Update)
 	mg.Deps(CrossBuild, CrossBuildXPack, CrossBuildGoDaemon)
```

Comment:
This one no longer needs to build any x-pack artifacts. And the `CrossBuildXPack` target can be removed.